plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -151,12 +151,23 @@ export async function createRouter(
   });
 
   router.get('/ratings/:entityRef', async (req, res) => {
+    const user = await identity.getIdentity({ request: req });
     const ratings = await dbHandler.getRatings(req.params.entityRef);
 
     const token = getBearerTokenFromAuthorizationHeader(
       req.header('authorization'),
     );
 
+    const entity = (
+      await catalogClient.getEntitiesByRefs(
+        {
+          entityRefs: [req.params.entityRef],
+          fields: ['spec.ownerRef'],
```

Comment:
I don't believe `spec.ownerRef` is a valid field - I assume you want `spec.owner`? If so, `spec.owner` is also not a standard field across all entities eg. User and Group entities don't have owners and even though rating users and team maybe not be a typical use case, this plugin current does allow for doing so